app/models/game.rb

```diff
@@ -1,5 +1,7 @@
 class Game < ActiveRecord::Base
-  has_many :reviews, dependent: :destroy
+  has_many :reviews, dependent: :destroy,
+  :after_add => :update_average,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.